sdk/mysql/Azure.ResourceManager.MySql/src/MySqlFlexibleServers/Custom/Models/OperationProgressResult.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.MySql.FlexibleServers.Models
+{
+    /// <summary> Represents Operation Results API Response. </summary>
+    public partial class OperationProgressResult : OperationStatusResult
```

Comment:
Can we reuse the existing https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/mysql/Azure.ResourceManager.MySql/src/MySqlFlexibleServers/Generated/Models/OperationStatusExtendedResult.cs instead?

Reply:
This is different that service will add discriminator for different operation.